drivers/LJPolymer.f90

```diff
@@ -0,0 +1,298 @@
+! This performs the calculations necessary to run a Lennard-Jones (LJ) 
+! simulation.
+! 
+! Copyright (C) 2013, Joshua More and Michele Ceriotti
```

Comment:
This seems out of date.

Reply:
We included the fast-forward Langevin in the i-pi v2 paper, so need to merge it soon into the main repository; we're doing this on i-pi-dev because i-pi/i-pi hasn't been cleaned up yet.

As to the out-of-date part, this certainly is, this was written by Mahdi Hijazi and I'll change it to reflect that